.drone.yml

```diff
@@ -492,6 +492,39 @@ steps:
     HOST: grafana-server
   image: cypress/included:9.5.1-node16.14.0-slim-chrome99-ff97
   name: end-to-end-tests-various-suite
+- commands:
+  - apt-get update && apt-get install git curl jq --yes
+  - cd ..
+  - git clone "https://$${GITHUB_TOKEN}@github.com/grafana/cloud-data-sources" --depth=1
+  - cd cloud-data-sources/e2e-scripts
+  - ./scripts/install-dependencies-ci.sh azure
+  - export PATH=$PATH:/root/.pulumi/bin
+  - ./scripts/azure-login.sh
+  - ./scripts/install.sh azure
+  - ./scripts/azure-pulumi-login.sh azure deploy ${DRONE_SOURCE_BRANCH}
+  - cd ../../grafana
+  - ./bin/build e2e-tests --port 3001 --suite cloud-plugins-suite --video false
+  - cd ../cloud-data-sources/e2e-scripts
+  - ./scripts/destroy.sh azure destroy ${DRONE_SOURCE_BRANCH}
```

Comment:
we need to make sure that this command runs even if the previous (`./bin/build e2e-tests`) fails, is there any safeguard for that?

Reply:
I will have a think on how to implement this, good catch!